security/pfSense-pkg-zeek/pkg-message

```diff
@@ -0,0 +1 @@
+Please visit Services -Zeek NSM menu to configure the package.
```

Comment:
Change to: "Navigate to Services > Zeek NSM in the WebGUI to configure the package"